zerver/openapi/zulip.yaml

```diff
@@ -1869,6 +1872,51 @@ paths:
                                     [["Denmark", "topic", 1594825442]],
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when when one of that user's
+                                configured user-topic relationships has changed.
+
+                                **Changes**: New in Zulip 6.0 (feature level 131).
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - user_topic
+                                stream_id:
+                                  type: integer
+                                  description: |
+                                    The ID of the stream to which the topic belongs.
+                                topic_name:
+                                  type: string
+                                  description: |
+                                    The name of the topic.
+                                last_updated:
+                                  type: integer
+                                  description: |
+                                    An integer UNIX timestamp representing when the user-topic
+                                    relationship was changed.
+                                visibility_policy:
+                                  type: integer
+                                  description: |
+                                    The type of visibility policy followed by the user-topic pair.
+
+                                    - 0 = None. Used in events to indicate that this topic now follows
+                                      its default user-topic configuration.
```

Comment:
This description isn't quite right -- there isn't really such a thing as a "default user-topic configuration". Empirically, see where @chrisbobbe was confused upon reading it: https://github.com/zulip/zulip-mobile/pull/5382#discussion_r875163549

Rather what this wants to say is that with this value, this user-topic pair no longer has a configuration of its own; its behavior follows the configuration it inherits from the stream.

Or more precisely: it has no *visibility policy* of its own, and behaves according to the user's visibility policy for the stream. At present the only content that a user-topic configuration has is the visibility policy, so the distinction between "user-topic configuration" and "visibility policy" currently doesn't make a difference in practice, but this API is designed so that there could be another field added here in the future and then it would.

Here's one version:

```suggestion
                                    - 0 = None. Used in events to indicate that the user no longer has
                                      a visibility policy specific to this topic. The topic will follow
                                      the user's policy for the stream (represented by `is_muted`
                                      on the subscription object.)
```